Automattic/woocommerce-payments
# Set ECE as Default for Express Checkout Buttons unless it was disabled.

Closes #8949

#### Changes proposed in this Pull Request

Set Express Checkout Elements (ECE) as the default for express checkout buttons unless it has been explicitly disabled by setting the option `_wcpay_feature_stripe_ece` to `0`.

#### Testing instructions

**Verify ECE is Enabled by Default:**

1. Run `npm run wp option delete _wcpay_feature_stripe_ece` to ensure the feature flag is not set.
2. Navigate to WooPayments Express Checkout settings. Confirm ECE is enabled if options for border radius for Google Pay and Apple Pay are available.

**Test Disabling ECE via Feature Flag:**

1. Run `npm run wp option update _wcpay_feature_stripe_ece 0` to disable ECE.
2. Check the same settings. There should not be options for border radius for Google Pay and Apple Pay, indicating ECE is disabled.

**Ensure Feature Flag Enables ECE When Set:**

1. Run `npm run wp option update _wcpay_feature_stripe_ece 1` to enable ECE.
2. Revisit the settings and confirm the presence of border radius options for Google Pay and Apple Pay, indicating ECE is enabled.

-------------------


**Post merge**
